tests/nnstreamer_filter_lua/unittest_filter_lua.cc

```diff
@@ -891,6 +910,53 @@ end
   sp->close (&prop, &data);
 }
 
+/**
+ * @brief Positive case with reload lua script with multiple types
+ */
+TEST (nnstreamerFilterLua, multipleTypes00)
+{
+  int ret;
+  void *data = NULL;
+  gchar *model;
+  GstTensorMemory input, output;
+  GstTensorFilterProperties prop;
+
+  output.size = input.size = sizeof (int64_t) * 1 * 2 * 2 * 1;
+
+  /* alloc input data without alignment */
+  input.data = g_malloc (input.size);
+  output.data = g_malloc (output.size);
+
+  memset (input.data, 0, input.size);
+
+  const GstTensorFilterFramework *sp = nnstreamer_filter_find ("lua");
+  EXPECT_NE (sp, nullptr);
+
+  for (int i = 0; i < _NNS_END; ++i) {
+    tensor_type ttype = (tensor_type) i;
+    model = g_strdup_printf ("\
+    inputTensorsInfo={num=1,dim={{1,2,2,1},},type={'%s',}} \
+    outputTensorsInfo={num=1,dim={{1,2,2,1},},type={'%s',}} \
+    function nnstreamer_invoke() \
+    for i=1,1*2*2*1 do output_tensor(1)[i] = input_tensor(1)[i] end \
+    end", gst_tensor_get_type_string (ttype), gst_tensor_get_type_string (ttype));
+
+    _SetFilterPropModelScript (&prop, "lua", model);
+    ret = sp->open (&prop, &data);
+    EXPECT_EQ (ret, 0);
+    EXPECT_NE (data, (void *) NULL);
+
+    ret = sp->invoke (NULL, NULL, data, &input, &output);
+    EXPECT_EQ (ret, 0);
+
+    g_free (model);
+  }
+
+  g_free (input.data);
+  g_free (output.data);
+  sp->close (&prop, &data);
+}
```

Comment:
Please add negative cases (```${name}_${num}_n```)

Reply:
I added a negative case for using invalid data type.